### PR TITLE
#50 Request meeting

### PR DESCRIPTION
**Pull request by vmtikk**

### CODE REVIEW

File: WebAPI/Controllers/WebComponentsController.cs [hunk 1]
```diff
@@ -65,7 +65,8 @@ public async Task<IActionResult> GetMeeting()
             }
 
             text = text.Replace("#--API_URL--#", apiUrl);
-
+            text = text.Replace("#--YEAR--#", year);
```
Comment (vijupa) [inline WebAPI/Controllers/WebComponentsController.cs:68]: #--MEETING_YEAR--#, #--MEETING-SEQUENCE_NUM--# could be more descriptive

File: WebAPI/StorageClient/StorageConnection.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+        public StorageConnection(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public HttpClient CreateConnection()
+        {
+            var connection = new HttpClient();
+            connection.BaseAddress = new Uri(_configuration["Storage:url"]);
```
Comment (jmsalonen) [inline WebAPI/StorageClient/StorageConnection.cs:20]: Shouldn't this be STORAGE_URL ?

File: WebAPI/Controllers/WebComponentsController.cs (hunk 1 above)
Comment (vmtikk) [reply WebAPI/Controllers/WebComponentsController.cs]: fixed

File: WebAPI/StorageClient/StorageConnection.cs (hunk 2 above)
Comment (vmtikk) [reply WebAPI/StorageClient/StorageConnection.cs]: fixed

**Review by vijupa — APPROVED** [review]
For me this is ok, but since it contains UI changes, lets wait for @koskelav approval before merging..

**Review by jmsalonen — APPROVED** [review]

**Pull request merged**